operations/configuration.md

```diff
@@ -120,27 +122,14 @@ Returns configuration of the enterprise and the client.
 | `Phone` | string | optional | Phone number of the enterprise. |
 | `LogoImageId` | string | required | Unique identifier of the enterprise logo image. |
 | `CoverImageId` | string | required | Unique identifier of the enterprise cover image. |
-| `Address` | [Address](#address) | required | Address of the enterprise. |
+| `AddressId` | string | required | Identifier of the [Address](addresses.md#account-address) of the enterprise. |
+| ~~`Address`~~ | ~~[Address](#address)~~ | ~~required~~ | ~~Address of the enterprise.~~ **Deprecated!** |
 | `Currencies` | array of [Accepted currency](#accepted-currency) | required | Currencies accepted by the enterprise. |
 | `Pricing` | string | required | [Pricing](#pricing) of the enterprise. |
 | `TaxPrecision` | string | optional | Tax precision used for financial calculations in the enterprise. If `null`, [Currency](currencies.md#currency) precision is used. |
 | `ExternalIdentifier` | string | optional, max 255 characters | Identifier of the enterprise from external system. |
 | `AccountingConfiguration` | [Accounting configuration](#accounting-configuration) | optional | Configuration information containg financial information about the property. |
 
-#### Address
-
-| Property | Type | Contract | Description |
-| :--- | :--- | :--- | :--- |
-| `Id` | string | required | Unique identifier of the address. |
-| `Line1` | string | optional | First line of the address. |
-| `Line2` | string | optional | Second line of the address. |
-| `City` | string | optional | The city. |
-| `PostalCode` | string | optional | Postal code. |
-| `CountryCode` | string | optional | ISO 3166-1 code of the [Country](countries.md#country). |
-| `CountrySubdivisionCode` | string | optional | ISO 3166-2 code of the administrative division, e.g. `DE-BW`. |
-| `Latitude` | number | optional | The latitude. |
-| `Longitude` | number | optional | The longitude. |
-
```

Comment:
I think this should stay and marked as deprecated 🤔 @MikeAdamsMews how do we handle this, I can't find page where we had the overall description?